src/api/RequestApi.ts

```diff
@@ -314,14 +409,25 @@ export const useCompleteRequest = (Id: number) => {
         queryClient.invalidateQueries(["requests", Id]);
 
         // Generate the email notification
-        sendInRequestCompleteEmail.mutate(request);
+        // TODO -- When implementing Out Request Email -- remove this typecheck, and update hook to send correct email
+        if (isInRequest(request)) {
+          sendRequestCompleteEmail.mutate(request);
+        }
       },
     }
   );
 };
 
+/** ENUM that defines which type of request In or Out Processing */
+export enum REQUEST_TYPES {
+  InRequest = 1,
+  OutRequest = 2,
+}
+
 // create IItem item to work with it internally
 export type IInRequest = {
+  /** Required - Whether it is In or Out Processing request */
+  reqType: REQUEST_TYPES.InRequest;
```

Comment:
While we don't NEED enums, I do find some value in code maintenace with them.  It provides a one stop place to update the "value" and ensure a spot within the code isn't missed in being updated.  However, since this is pretty much a fixed value that won't be updated, we could switch to a hardcoded value through the various code files.  I'll look at updating in the commit after I post my current commit.